tests/rule_based_profiler/alpha/test_profiler_user_workflows.py

```diff
@@ -306,3 +307,99 @@ def test_bobster_profiler_user_workflow_multi_batch_row_count_range_rule_bootstr
+
+    # Load profiler configs & loop (run tests for each one)
+    yaml_config: str = quentin_columnar_table_multi_batch["profiler_config"]
+
+    # Instantiate Profiler
+    profiler_config: CommentedMap = yaml.load(yaml_config)
+
+    # Roundtrip through schema validation to remove any illegal fields add/or restore any missing fields.
+    deserialized_config: dict = ruleBasedProfilerConfigSchema.load(profiler_config)
+    serialized_config: dict = ruleBasedProfilerConfigSchema.dump(deserialized_config)
+
+    # `class_name`/`module_name` are generally consumed through `instantiate_class_from_config`
+    # so we need to manually remove those values if we wish to use the **kwargs instantiation pattern
+    serialized_config.pop("class_name")
+    serialized_config.pop("module_name")
+
+    profiler: RuleBasedProfiler = RuleBasedProfiler(
+        **serialized_config,
+        data_context=data_context,
+    )
+
+    expectation_suite: ExpectationSuite = profiler.run(
+        expectation_suite_name=quentin_columnar_table_multi_batch["test_configuration"][
+            "expectation_suite_name"
+        ],
+    )
+
+    expectation_configuration_dict: dict
+    column_name: str
+    expectation_kwargs: dict
+    expect_column_quantile_values_to_be_between_expectation_configurations_kwargs_dict: Dict[
+        str, dict
+    ] = {
+        expectation_configuration_dict["kwargs"][
+            "column"
+        ]: expectation_configuration_dict["kwargs"]
+        for expectation_configuration_dict in expectation_suite.to_json_dict()[
+            "expectations"
+        ]
+    }
+    expect_column_quantile_values_to_be_between_expectation_configurations_value_ranges_by_column: Dict[
+        str, List[List[Number]]
+    ] = {
+        column_name: expectation_kwargs["value_ranges"]
+        for column_name, expectation_kwargs in expect_column_quantile_values_to_be_between_expectation_configurations_kwargs_dict.items()
+    }
+
+    assert (
+        expect_column_quantile_values_to_be_between_expectation_configurations_value_ranges_by_column[
+            "tolls_amount"
+        ]
+        == quentin_columnar_table_multi_batch["test_configuration"][
+            "expect_column_quantile_values_to_be_between_quantile_ranges_by_column"
+        ]["tolls_amount"]
+    )
+
+    rtol: float = 1.0e-7
+    atol: float = 5.0e-2
+
+    value_range: List[Number]
+    paired_quantiles: zip
+    column_quantiles: List[List[Number]]
+    idx: int
+    for (
+        column_name,
+        column_quantiles,
+    ) in (
+        expect_column_quantile_values_to_be_between_expectation_configurations_value_ranges_by_column.items()
+    ):
+        paired_quantiles = zip(
+            column_quantiles,
+            quentin_columnar_table_multi_batch["test_configuration"][
+                "expect_column_quantile_values_to_be_between_quantile_ranges_by_column"
+            ][column_name],
+        )
```

Comment:
Nitpick - similar point as above but I think retrieving values from nested dicts like this is difficult to read.

What are your thoughts on this?
```python
q = quentin_columnar_table_multi_batch["test_configuration"]["expect..."]

...

paired_quantiles = zip(
    column_quantiles,
    q[column_name],
)
```

Reply:
Since the `quentin_columnar_table_multi_batch` is from a fixture, I think it is OK to leave it as is (for now).  Thanks.